ERCS/erc-7548.md

```diff
@@ -0,0 +1,180 @@
+---
+eip: 7548
+title: Open IP Protocol
+description: A protocol that enables users to remix NFTs and generate new NFT derivative works, while their relationships can be traced on chain.
+author: Combo <combo@1combo.io>, Saitama (@saitama2009), CT29 <CT29@1combo.io>, Luigi <luigi@1combo.io>
+discussions-to: https://ethereum-magicians.org/t/draft-open-ip-protocol/16373
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-10-31
+requires: 165, 721
+---
+
+## Abstract
+
+This proposal aims to standardize how to remix multiple existing IPs and create a new IP in a decentralized way. It extends the [ERC-721](./eip-721.md) standard, enabling users to remix multiple existing NFTs and create a new NFT derivative work, while their relationships can be traced on the blockchain. 
```

Comment:
Another thing to note about the abstract: it should give a high level technical summary of the whole proposal. How do you accomplish your goal?